wtfis/models/abuseipdb.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from typing import Optional, List
+
+from pydantic import BaseModel, Field, RootModel
+
+
+class AbuseIpDb(BaseModel):
+    ip_address: str = Field(alias="ipAddress")
+    is_public: Optional[bool] = Field(None, alias="isPublic")
+    ip_version: Optional[int] = Field(None, alias="numDistinctUsers")
+    is_whitelisted: Optional[bool] = Field(None, alias="ipVersion")
+    abuse_confidence_score: int = Field(alias="abuseConfidenceScore")
+    country_code: Optional[str] = Field(None, alias="countryCode")
+    country_name: Optional[str] = Field(None, alias="countryName")
+    usage_type: Optional[str] = Field(None, alias="usageType")
+    isp: str
+    domain: Optional[str] = None
+    hostnames: Optional[List[str]] = None
+    is_tor: Optional[bool] = Field(None, alias="isTor")
+    total_reports: Optional[int] = Field(None, alias="totalReports")
+    num_distinct_users: Optional[int] = Field(None, alias="numDistinctUsers")
+    last_reported_at: Optional[str] = Field(None, alias="lastReportedAt")
+    reports: Optional[List] = None
```

Comment:
Just checked the [docs](https://docs.abuseipdb.com/#check-endpoint) and it looks like this is a list of dicts:

```
      "reports": [
        {
          "reportedAt": "2018-12-20T20:55:14+00:00",
          "comment": "Dec 20 20:55:14 srv206 sshd[13937]: Invalid user oracle from 118.25.6.39",
          "categories": [
            18,
            22
          ],
          "reporterId": 1,
          "reporterCountryCode": "US",
          "reporterCountryName": "United States"
        }
      ]
```
Normally, I would create a new model to represent the dict, but if you're not using this field at all then you can just remove this line.

Reply:
Well, I intended to use it but the report detail will not be used in a calculation or displayed at all. I will just remove it. For the sake of simplification, I can just remove other unused fields too then. That'd simplify the model.

Reply:
Removed reports but kept others which may be valuable. But YAGNI tells me I should remove others. Not sure.

Reply:
All good, we can keep the rest as is.